ocfweb/docs/docs/staff/procedures/accounts/renaming.md

```diff
@@ -0,0 +1,14 @@
+[[!meta title="Rename an account"]]
+
+We should sparingly rename an user account, and only do so when it is
```

Comment:
```suggestion
We should sparingly rename a user account, and only do so when it is
```
Interestingly, in English, there is a rule that if a `u` making a `y` sound, you use `a` not `an`. Wtf English.